### ISSUE TITLE
Setup tensorboard to monitor training loss and validation loss

### ISSUE THREAD

**Issue by varun-suresh** [varun-suresh/ml-practice#5]
Plot training and validation loss as training progresses.
